Fix typos and small issues with the tutorial (#125)
ESMValGroup/ESMValTool_Tutorial: _episodes/06-debugging.md

> ## ESMValTool can’t locate the data
>

> > You are assisting a colleague with ESMValTool. The colleague changes the

Reviewer: ```suggestion
> You are assisting a colleague with ESMValTool. The colleague replaces the
```

> > ``datasets/project`` entry to ``CMIP6`` and runs the recipe. However, ESMValTool

Reviewer: ```suggestion
> "CMIP5" in ``project: CMIP5`` to ``CMIP6`` and runs the recipe. However, ESMValTool
```

The script path should be relative to ``diag_scripts`` directory. It means that
the script ``diagnostic_timeseries.py`` is located in
``<path_to_esmvaltool>/diag_scripts/ocean/``. Alternatively, the script path can
be an absolute path. To examine this, we create a small python script `hello.py`:

```python
with open('hello_world.txt', 'w') as f:
    print('Fello from ESMValTool!', file=f)
```

> Change and run the recipe:

Reviewer: Nice example. However, this section aims to show the difference between the absolute and relative path of the diagnostic script. "creating your own diagnostic" is another episode in the advanced part of the tutorial that would be developed in the near future.

> Change and run the recipe:

Author: It was the only thing I could get to work with my limited knowledge 3 weeks ago 😅

The absolute path you are referring to here is very deep in miniconda, so I didn't think the instruction was very useful. The path for me is nested somewhere in: `~/miniconda3/envs/esmvaltool/lib/python3.8/site-packages/`. I think it is bad practice to recommend people to go looking around there. What do you think about adding a link to the diag script from the ESMValTool directory and running them from the working directory?

```yaml
40        scripts:
41          timeseries_diag:

> 42            script: ~/esmvaltool_tutorial/hello.py

Reviewer: ```suggestion
42            script: <path_to_esmvaltool>/diag_scripts/ocean/diagnostic_timeseries.py
```

```

~~~bash
  esmvaltool run recipe_example.yml
~~~

> Now examine `./esmvaltool_output/recipe_example_#_#/run/diag_timeseries_temperature/timeseries_diag/` to see if it worked! Of course, this is a simple example, but it shows how you can customize diagnostic scripts or even make your own. Have a look through the scripts available in the [ESMValTool repository](https://github.com/ESMValGroup/ESMValTool/blob/master/esmvaltool/diag_scripts/).
> 

Reviewer: ```suggestion
Now examine `./esmvaltool_output/recipe_example_#_#/run/diag_timeseries_temperature/timeseries_diag/` to see if it worked! Of course, this is a simple example, but it shows how you can customize diagnostic scripts or even make your own.

```

Reviewer: "creating your own diagnostic" is another episode in the advanced part of the tutorial that would be developed in the near future. Also, "Available recipe and diagnostic scripts" are already mentioned in the next section.
